Sources/SpeziViews/Views/Text/MarkdownView.swift

```diff
@@ -55,6 +79,7 @@ public struct MarkdownView: View {
             }
         }
             .task {
+                state = .processing
                 markdown = await asyncMarkdown()
```

Comment:
Looking at the state here: I think it might make sense to annotated the `markdownString` property with `@State` and make it optional. As a result, the `markdown` property could then be removed.

If we do that, we could use `AttributedString(String(localized: "MARKDOWN_LOADING_ERROR", bundle: .module))` as the value for `markdownString` in case we have an error and then do all the markdown creation that we currently do in `markdownString`, state handling, management in this task closure (or a private function that we call here).

You can then also use `MainActor.run` to switch the context to the main actor.